core/appeal/service.go

```diff
@@ -1,6 +1,16 @@
+//go:generate mockery --name=repository --exported
```

Comment:
We can use `--with-expecter` here so we have more strongly typed mock functions https://lens.golabs.io/graph/d/nWve-U2Gz/elasticsearch-global-search-platform?orgId=1&refresh=30s&var-datasource=tardis&var-environment=production&var-cluster=global-search-production&var-host=All

Reply:
nice, will check if we're gonna need this now